components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -129,7 +134,8 @@ public Response applicationsGet(String groupId, String query, String sortBy, Str
         ApplicationListDTO applicationListDTO = new ApplicationListDTO();
 
         String username = RestApiCommonUtil.getLoggedInUsername();
-        
+        String organization = getOrganization(messageContext);
```

Comment:
I think we can make this method a utility method in a common class and use in other places too.

Reply:
fixed from 0e19391